open-hackathon-client/src/client/static/js/views/oh.user.profile.js

```diff
@@ -123,6 +123,18 @@
             li_join.text('TA参与的活动');
             $('#my_events').detach();
         }
+
+        //choose which panel to open by urlParam
+        var active_panel = $.getUrlParam('active_panel') || 0;
+        if (active_panel == 1) {
+            $('#my_register_events_tab').trigger('click');
```

Comment:
Do we have to trigger `click` event? Can we just execute the code hooked on `click` event?